<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pic_Widget</class>
 <widget class="QWidget" name="pic_Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="img_viewer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>240</width>
     <height>320</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>240</width>
     <height>320</height>
    </size>
   </property>
   <property name="whatsThis">
    <string comment="img_viewer" extracomment="img_viewer"/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="refStardand">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>240</width>
     <height>320</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="img_x1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>240</width>
     <height>320</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="img_x2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>250</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>240</width>
     <height>320</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="img_y1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>240</width>
     <height>320</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="img_y2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>240</width>
     <height>320</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="img_z1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>630</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>240</width>
     <height>320</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="img_z2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>630</y>
     <width>120</width>
     <height>160</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>240</width>
     <height>320</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yaw">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pitch">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_roll">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_yaw">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>yaw</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_roll">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>roll</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_pitch">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>pitch</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_yaw">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>370</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>-90</number>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_HoriSlider_yaw">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>330</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>yaw</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit__HoriSlider_yaw">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>330</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_HoriSlider_pitch">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>460</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>pitch</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit__HoriSlider_pitch">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>460</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_pitch">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>500</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>-90</number>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_HoriSlider_roll">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>600</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>roll</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit__HoriSlider_roll">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>600</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_roll">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>640</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>-90</number>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
